container/lxd/network.go

```diff
@@ -239,7 +238,7 @@ func (s *Server) verifyNICsWithConfigFile(nics map[string]device, reader func(st
 			continue
 		}
 
-		logger.Infof("found usable network device %q with parent %q", name, netName)
+		logger.Debugf("found usable network device %q with parent %q", name, netName)
```

Comment:
This output doesn't look to have changed in frequency, so this should be Tracef like on line 215 (I missed it before). In any case, they should both be the same.